blocks/carousel/carousel.css

```diff
@@ -459,6 +492,36 @@
 @media (width >= 900px) {
   .carousel.vertical {
     height: 500px;
+    padding-left: 50px;
+    padding-right: 40px;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  @media (width >= 1200px) {
+    .carousel.vertical {
+      height: 600px;
+      width: 1100px;
+    }
+  }
+
+  @media (width >= 1400px) {
+    .carousel.vertical {
+      height: 600px;
+      width: 1280px;
+      padding: 0;
+      margin-left: -25px;
+      margin-right: 0;
+    }
+
+    /* Reset padding for all vertical carousel variants */
+    .carousel.vertical.hero-banner,
+    .carousel.vertical.scrollable,
+    .carousel.vertical.news-slider,
+    .carousel.vertical.featured-projects {
+  padding: unset;
+  margin: unset;
+}
```

Comment:
if you unset in one smaller viewport, it applies to larger viewports too, please remove unrequired references